Content.Client/Medical/CrewMonitoring/CrewMonitoringWindow.xaml.cs

```diff
@@ -67,12 +80,39 @@ public void ShowSensors(List<SuitSensorStatus> stSensors, Vector2 worldPosition,
                 // add users positions
                 // format: (x, y)
                 var box = GetPositionBox(sensor.Coordinates, worldPosition, snap, precision);
+
                 SensorsTable.AddChild(box);
                 _rowsContent.Add(box);
+
+                if (sensor.Coordinates != null && NavMap.Visible)
+                {
+                    NavMap.TrackedCoordinates.Add(sensor.Coordinates.Value, (true, Color.Red));
+                    nameLabel.MouseFilter = MouseFilterMode.Stop;
+
+                    // Hide all others upon mouseover.
+                    nameLabel.OnMouseEntered += args =>
```

Comment:
Imo current UX with mouse hoover is a bit confusing. Maybe use toggles? Change selected dot color to green or something?

Reply:
Crew monitoring window itself needs a redesign so might leave it to later.